source/tutorials/nvidia.rst

```diff
@@ -482,6 +482,15 @@ try one of the solutions below:
      EndSection   
      EOF
 
+Loading saved settings
+==================
+Settings made through the nvidia-settings control panel persist until they are unloaded. You may save your settings under the nvidia-settings Configuration item by clicking the button labeled "Save Current Configuration" or if the "write-config" option is enabled in ~/.nvidia-settings-rc settings are saved when the application closes (this behavior is default).
+
+However saved settings are not loaded by default unless you launch nvidia-settings.  To make them persistent we must call nvidia-settings at the begginging of the session.  To do this:
+
+   .. code:: bash
+   echo "nvidia-settings --load-config-only" >> ~/.xinitrc
```

Comment:
Indent code-block so "e" in echo aligns with "c" in code above it. This will fix the build error. 

```suggestion

 .. code:: bash
    echo "nvidia-settings --load-config-only" >> ~/.xinitrc

    ```